### ISSUE TITLE
Support for different keyboards

### ISSUE THREAD

**Issue by spatialfree** [StereoKit/StereoKit#486]
Make flatscreen simulation input accessible to a host of custom keyboards/layouts and languages. By mapping the keys based on their Physical Location instead of Value.

I've noticed that Unity's implemented this in their newer [input system](https://forum.unity.com/threads/new-input-system-which-keyboard-layouts-are-supported.912074/). As well as a variety of games/apps seem to work this way as well, but I can't seem to find the common thread just yet... One easy way to test for this is to install the [colemak](https://colemak.com/) layout and toggle it on-off.

**Comment by maluoi** [comment]
Could you elaborate on this a little more? StereoKit does support Unicode input (try changing your keyboard to say, Cyrillic), and input text into a `UI.Input`. If you're referring to `Input.Key`, these map directly to [virtual keys](https://learn.microsoft.com/en-us/windows/win32/inputdev/virtual-key-codes) which are different from text input.

If you're looking to read text input rather than keys, you can access these through [`Input.TextConsume`](https://stereokit.net/Pages/StereoKit/Input/TextConsume.html)/`TextReset`!

**Comment by spatialfree** [comment]
Thanks! That link helped me find the correct relevant term: "[Scan Codes](https://learn.microsoft.com/en-us/windows/win32/inputdev/about-keyboard-input#scan-codes)"

**Comment by maluoi** [comment]
Ah ok, does this mean you're wanting the current `Key`s to behave less like virtual keys, and more like scan codes?

**Comment by spatialfree** [comment]
Yeah! For when you are flying around on desktop, the key position matters more than the "value".
